pkg/libvmi/storage.go

```diff
@@ -40,6 +40,14 @@ func WithContainerDisk(diskName, imageName string) Option {
 	}
 }
 
+// WithContainerSATADisk specifies the disk name and the name of the container image to be used.
+func WithContainerSATADisk(diskName, imageName string) Option {
```

Comment:
So API wise I am not sure. Thinking maybe optional `containerDiskOpts` would make more sense in `WithContainerDisk`?
cc @EdDev